File: decompiled/sources/com/hjq/http/ssl/UnSafeHostnameVerifier.java
package com.hjq.http.ssl;

import javax.net.ssl.HostnameVerifier;
import javax.net.ssl.SSLSession;

public final class UnSafeHostnameVerifier implements HostnameVerifier {
    public boolean verify(String str, SSLSession sSLSession) {
        return true;
    }
}

File: decompiled/sources/com/hjq/http/ssl/UnSafeHostnameVerifier.kt
package com.hjq.http.ssl

import javax.net.ssl.HostnameVerifier
import javax.net.ssl.SSLSession

class UnSafeHostnameVerifier : HostnameVerifier {
    override fun verify(str: String?, sSLSession: SSLSession?): Boolean {
        return true
    }
}
